tests/Twig/Extension/TemplateRegistryExtensionTest.php

```diff
@@ -56,16 +52,6 @@ protected function setUp(): void
         );
     }
 
-    public function getFunctionsTest(): void
```

Comment:
maybe this function should get a test as prefix so it is run as a test?

Reply:
The test wasnt working because the instance of twigfunctions were not the same.

Reply:
Imo we could modify a bit that test to assert N elements in the array.

Reply:
Done